README.md

```diff
@@ -23,15 +23,15 @@ $ pip install corva-sdk
 
 ## App types
 
-There are three app types, that you can build:
+There are three app types that you can build:
 
-1. `stream` - works with a real-time data
+1. `stream` - works with real-time data
 2. `scheduled` - works with data at defined schedules/intervals
    (e.g. once an hour)
 3. `task` - works with data on-demand
 
-**Note**: it is recommended to use type hints like in examples below, so that editors and tools can give you better
-support.
+**Note**: It is recommended to use type hints like those in the examples below. <br>
```

Comment:
I don't think we should use <br>s in a markdown file

Reply:
ok